tzcode/PKGBUILD

```diff
@@ -5,6 +5,7 @@ _ver=2018c
 # use a pacman compatible version scheme
 pkgver=${_ver/[a-z]/.${_ver//[0-9.]/}}
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `tzcode`. Therefore this change should be unnecessary.